_features/offline-camp.md

```diff
@@ -9,31 +9,25 @@ img-thumb:  offline-camp-campfire.jpeg #offlinecamp-logo-thumb.jpg
 strategies: 
   - offline-first
 essentials:
+  Offline Camp:  http://offlinefirst.org/camp/  
   Offline Camp on Medium: https://medium.com/offline-camp
-  Offline Camp Calendar:  http://offlinefirst.org/camp/
-  Offline First!: http://offlinefirst.org/camp/
+  Offline Camp on YouTube: https://www.youtube.com/OfflineFirst
+  Offline First: http://offlinefirst.org/
 ---
 
-this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph. this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph.
+Offline First is about more than just technology; it’s about community. That’s why we’re so happy to be foundational sponsors of the community-driven unconference known as [Offline Camp](http://www.offlinefirst.org/camp). Each retreat takes place over the course of a long weekend in a remote location, gathering developers, UX professionals, and business leaders to discuss the opportunities and challenges around [Offline First](http://www.offlinefirst.org). The unconference format allows us to tailor discussions to the interests of the 20 to 30 campers who attend each camp, and the idyllic settings and communal living encourage a genuine sense of [community built around a campfire](https://medium.com/offline-camp/crafting-community-events-that-leave-them-wanting-smore-902974cff4d4). Our team has sponsored and co-organized each of the first three camps, from the Catskills to California to Berlin, and we’re thrilled to be in the company of other stand-out sponsors such as Google Cloud Platform, Make&Model Inc, Neighbourhoodie, simplabs, and YLD.
```

Comment:
Where you say, "Our team has sponsored…" I would consider instead saying something like, "The IBM Watson Data Lab team has sponsored…"